indico/modules/rb_new/client/js/modules/bookRoom/BookRoom.jsx

```diff
@@ -309,78 +309,26 @@ class BookRoom extends React.Component {
 
         return (
             <>
-                <Header as="h2">
-                    <Translate>Rooms that you might be interested in</Translate>
+                <Divider horizontal>
+                    <Icon name="magic" style={{fontSize: '2em'}} circular />
```

Comment:
not covered by any of the predefined sizes?

Reply:
Actually, setting the size doesn't change much because somehow the icon limits itself to the height of the divider. But I will move the override to the SCSS module.